src/main/java/org/alfresco/utility/data/DataContent.java

```diff
@@ -732,29 +732,43 @@ public FileModel getCustomModel()
     {
         return customModel;
     }
-    
-
 
-    public ContentStream getContentStream(String fileName, String content) throws Exception
+    /**
+     * Create a content stream from some content.
+     *
+     * @param fileName A name for the content stream.
+     * @param content The content.
+     * @return The stream.
+     * @throws IORuntimeException if there is an issue creating the stream.
+     */
+    public ContentStream getContentStream(String fileName, String content) throws IORuntimeException

```

Comment:
No need to put the runtime excpetion in the method signature

Reply:
I'm happy to remove this, but I was trying to advertise it to the caller, in case they wanted to use it.  Do you think leaving it on the javadoc is enough?

Reply:
I think leaving it in the javadoc is enough but I've not a strong opinion about that.